builtin/commit-graph.c

```diff
@@ -146,7 +146,7 @@ static int graph_write(int argc, const char **argv)
 		flags |= COMMIT_GRAPH_WRITE_SPLIT;
 	if (opts.progress)
 		flags |= COMMIT_GRAPH_WRITE_PROGRESS;
-	if (opts.enable_bloom_filters)
+	if (opts.enable_bloom_filters || git_env_bool(GIT_TEST_COMMIT_GRAPH_BLOOM_FILTERS, 0))
```

Comment:
commit message:

1. it starts with `GIT_TEST_`.

2. be sure to include something about how the test suite passes when enabling this with `GIT_TEST_COMMIT_GRAPH=1` and `GIT_TEST_COMMIT_GRAPH_BLOOM_FILTERS=1` the test suite passes.

3. [Update the CI builds to turn this on](https://github.com/git/git/blob/12029dc57db23baef008e77db1909367599210ee/ci/run-build-and-tests.sh#L21)